aten/src/ATen/ExpandUtils.cpp

```diff
@@ -8,12 +8,12 @@ std::vector<int64_t> infer_size(IntList a, IntList b) {
   ptrdiff_t ndim = dimsA > dimsB ? dimsA : dimsB;
```

Comment:
I don't think this needs to be ptrdiff_t...there are literally no pointers involved here.